README.md

```diff
@@ -317,14 +317,18 @@ To run a particular test or subset of tests
 If database tests are failing because of a `permission denied` error, give your postgres user permissions to create a database.
 In the postgres shell, run the following as a superuser: `ALTER USER commcarehq CREATEDB;`
 
-### REUSE DB
-To avoid having to run the databse setup for each test run you can specify the `REUSE_DB` environment variable
- which will use an existing test database if one exists:
+### REUSE_DB and --reusedb
+To avoid having to run the databse setup for each test run
+ you can specify the `REUSE_DB` environment variable
+ or use the --reusedb argument
+ to use an existing test database if one exists:
 
     $ REUSE_DB=1 ./manage.py test corehq.apps.app_manager
+    $ ./manage.py test corehq.apps.app_manager --reusedb=1
```

Comment:
i don't think it's a big deal but i find it a little confusing / hard to tell that this is intended to represent two different ways to do the same thing as opposed to two commands you have to run in a row. I imagine it would help if you separated them with an "or" or something.

Reply:
Ah, good feedback. I'll clarify that.